core/src/com/nukesz/game/GameScreen.java

```diff
@@ -148,6 +153,12 @@ private void draw() {
                     layout.width) / 2, (viewport.getWorldHeight() - layout.height) / 2);
 
         }
+        if (state == State.INITIAL) {
+            layout.setText(bitmapFont, GAME_START_TEXT);
+            bitmapFont.draw(batch, GAME_START_TEXT, (viewport.getWorldWidth() -
+                    layout.width) / 2, (viewport.getWorldHeight() - layout.height) / 2);
+
+        }
```

Comment:
This whole "if" could be a method and used with GAME_START_TEXT or GAME_OVER_TEXT parameter to avoid duplication.